nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -105,3 +105,10 @@ notification.max.attempts=5
 # The first curator connection issue is logged as ERROR, for example when NiFi cannot connect to one of the Zookeeper nodes.
 # Additional connection issues are logged as DEBUG until the connection is restored.
 java.arg.curator.supress.excessive.logs=-Dcurator-log-only-first-connection-issue-as-error-level=true
+
+# Diagnostics
+nifi.diag.allowed=true
```

Comment:
Creating diagnostics for everyone by default seems the wrong change to make - I'd default this to `false` otherwise existing installations will suddenly find their filesystems filling up with unexpected files whenever their NiFi instance restarts (the default settings also allow *many* files to be retained, so I could easily see people getting a bit confused/frustrated with the fact that diagnostics files are filling up their filesystems)

If this is left as `true` I'd greatly reduce the default `max` retention settings (e.g. keep up to the last 2 or 10 files)

Reply:
(As a general note, I'd prefer comments to be left for the original author to Resolve, otherwise I've got to work back through all comments to see what it was I previously identified as needing work)

However, happy with the change to `false` by default, but this feature seems like something that should be included in the Admin Guide, particularly if simply setting `nifi.diag.allowed` to `true` without changing anything else would effectively allow the system to keep generating diganostic files until the disk filled up - that's something administrators should probably think about changing when enabling this functionality

Reply:
Admin Guide updates LGTM :+1: